src/en/users-creating.md

```diff
@@ -65,13 +65,34 @@ juju register
 MD4TA2ppbTAVExMxMC4xOTUuMTAuMTMwOjE3MDcwBCDENImZEDvYcWLU1USgaUJQXhWR98JNLWcbS0-MiW6NiQA=
 ```
 
+The user will then be prompted to enter and confirm a password, and also supply a unique name for the controller.
+
+### Lost tokens
+
+If a situation occurs where the registering token fails to work for some reason, or is
+mislaid, a new token can be generated by the admin user with the `change-user-password` command.
```

Comment:
I know we use en-GB but I've never heard of 'mislaid' before. Maybe consider something more direct like 'lost'.

Reply:
hmm. ok. But mislay is also in all the US dictionaries . Maybe it is just Canada that doesn't use it ;)